packages/netsuite-adapter/src/client/client.ts

```diff
@@ -462,28 +479,11 @@ export default class NetsuiteClient {
           .toArray()
     ).flatten(true).toArray()
 
-
-    const failedTypes = new Set<string>()
     log.debug('Fetching custom objects one by one')
     await withLimitedConcurrency( // limit the number of open promises
-      idsChunks.map(({ type, ids, index, total }: ObjectsChunk) => async () => {
-        if (failedTypes.has(type)) {
-          return
-        }
-        try {
-          log.debug('Starting to fetch chunk %d/%d with %d objects of type: %s', index, total, ids.length, type)
-          await this.runImportObjectsCommand(executor, type, ids.join(' '),
-            this.fetchTypeTimeoutInMinutes)
-          log.debug('Fetched chunk %d/%d with %d objects of type: %s', index, total, ids.length, type)
-        } catch (e) {
-          log.warn('Failed to fetch chunk %d/%d with %d objects of type: %s', index, total, ids.length, type)
-          log.warn(e)
-          failedTypes.add(type)
-        }
-      }),
+      idsChunks.map(idsChunk => async () => importObjectsChunk(idsChunk)),
```

Comment:
nit: (no need for `async` when there is no `await`)
```suggestion
      idsChunks.map(idsChunk => () => importObjectsChunk(idsChunk)),
```